modules/core/src/main/java/org/apache/ignite/spi/communication/tcp/TcpCommunicationSpi.java

```diff
@@ -3128,7 +3128,7 @@ else if (log.isDebugEnabled())
 
             if (time > CONNECTION_ESTABLISH_THRESHOLD_MS) {
                 if (log.isInfoEnabled())
```

Comment:
We shouldn't check "INFO" level if we now log to "WARNING" level

Reply:
I propose following message text
`Slow TCP client created[client=YYY,threshold=XXX,duration=ZZZms]`

What do you think?

Reply:
You're right.